src/docs/content/observability/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Observability"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+weight: 500
+showChildren: true
+---
+
+## Introduction
+
+Observability helps to understand and explain the state of your systems using a combination of logs, metrics and traces. It helps bringing better visibility into systems.
+
+Observability on OCI has a much broader scope, Read the recent [announcement](https://blogs.oracle.com/cloud-infrastructure/announcing-the-general-availability-of-oracle-cloud-infrastructure-logging) for more information.
```

Comment:
While this is not strictly enforced in MuShop, generally it is recommended to write "Oracle Cloud Infrastructure" completely vs "OCI". 

Consider removing "recent"